src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcListWithActions.scala.html

```diff
@@ -24,6 +24,11 @@
 @(params: ListWithActions)
 @import params._
 
+@*
+this file is white space sensitive, the below lines should not be changed
+44
+54 - 56
+*@
```

Comment:
Similarly - not sure if this comment is needed? I wonder if there is a something else we can do about testing the rendered HTML rather than a comment?

Reply:
I can remove the comments but there is no quick fix for testing whitespace or lack of them, we may need to think about a future enhancement to how we test twirl and nunjucks output where we can take spacing etc into consideration.